cpp/src/barretenberg/plonk/composer/composer_helper/composer_helper_lib.cpp

```diff
@@ -49,7 +49,8 @@ std::shared_ptr<plonk::verification_key> compute_verification_key_common(
     auto circuit_verification_key = std::make_shared<plonk::verification_key>(
         proving_key->circuit_size, proving_key->num_public_inputs, vrs, proving_key->composer_type);
     // TODO(kesha): Dirty hack for now. Need to actually make commitment-agnositc
-    auto commitment_key = proof_system::honk::pcs::kzg::CommitmentKey(proving_key->circuit_size, "../srs_db/ignition");
+    auto commitment_key =
```

Comment:
What changed here?

Reply:
The `CommitmentKey` is inside Params now (see honk/pcs/commitment_key.hpp)